### PR TITLE
docs: link to GetEnvoy.io for pre-built binaries

### LINKED ISSUE #5668: Provide raw Envoy binaries outside of Docker
**Issue Template**

*Title*: *Where can we download the standalone Envoy Binaries*

*Description*:
>We want to use Envoy in a standalone environment(windows subsystem for linux) without docker, can you please let us know 
1) where can we download the Envoy binaries
2) How to get started using Envoy binaries in a docker less environment, i am assuming it will similar except for the docker part.

[optional *Relevant Links*:]
>Any extra documentation required to understand the issue.

**Bug Template**

*Title*: *One line description*

*Description*:
>What issue is being seen? Describe what should be happening instead of
the bug, for example: Envoy should not crash, the expected value isn't
returned, etc.

*Repro steps*:
> Include sample requests, environment, etc. All data and inputs
required to reproduce the bug.

>**Note**: The [Envoy_collect tool](https://github.com/envoyproxy/envoy/blob/master/tools/envoy_collect/README.md)
gathers a tarball with debug logs, config and the following admin
endpoints: /stats, /clusters and /server_info. Please note if there are
privacy concerns, sanitize the data prior to sharing the tarball/pasting.

*Admin and Stats Output*:
>Include the admin output for the following endpoints: /stats,
/clusters, /routes, /server_info. For more information, refer to the
[admin endpoint documentation.](https://www.envoyproxy.io/docs/envoy/latest/operations/admin)

>**Note**: If there are privacy concerns, sanitize the data prior to
sharing.

*Config*:
>Include the config used to configure Envoy.

*Logs*:
>Include the access logs and the Envoy logs.

>**Note**: If there are privacy concerns, sanitize the data prior to
sharing.

*Call Stack*:
> If the Envoy binary is crashing, a call stack is **required**.
Please refer to the [Bazel Stack trace documentation](https://github.com/envoyproxy/envoy/tree/master/bazel#stack-trace-symbol-resolution).

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

Description:
Risk Level:
Testing:
Docs Changes: Docs only
Release Notes:
Fixes #5668

### CODE REVIEW

File: docs/root/install/building.rst [hunk 1]
```diff
@@ -68,6 +68,9 @@ tests.
   organizations track and deploy master in production. We encourage you to do the same so that
   issues can be reported as early as possible in the development process.
 
+There are packaged Envoy pre-built binaries in rpm, deb, and Homebrew packages are available via
```
Comment (mattklein123) [inline docs/root/install/building.rst]:
```suggestion
Packaged Envoy pre-built binaries for a variety of platforms are available via
```

File: docs/root/install/building.rst [hunk 2]
```diff
@@ -68,6 +68,9 @@ tests.
...
+`GetEnvoy.io <https://www.getenvoy.io/>`_.
+
 We will consider producing additional binary types depending on community interest in helping with
 CI, packaging, etc. Please open an `issue <https://github.com/envoyproxy/envoy/issues>`_ in GitHub
```
Comment (mattklein123) [inline docs/root/install/building.rst]: Maybe add a GH link to getenvoy here and tell people to open issue against that repo for prebuilt binaries for different platforms? WDYT?

Comment (lizan) [reply docs/root/install/building.rst]: SGTM, done

File: docs/root/start/start.rst [hunk 3]
```diff
@@ -5,9 +5,9 @@ Getting Started
 
 This section gets you started with a very simple configuration and provides some example configurations.
 
-Envoy does not currently provide separate pre-built binaries, but does provide Docker images. This is
-the fastest way to get started using Envoy. Should you wish to use Envoy outside of a
-Docker container, you will need to :ref:`build it <building>`.
+Envoy provides official :ref:`pre-built Docker images <install_binaries>` and other pre-built binaries
```
[the remark this answers is not in the document]
Comment (lizan) [reply docs/root/start/start.rst]: done.

**Pull request merged**